### ISSUE TITLE
Early turning problem

### ISSUE THREAD

**Issue by jackspp** [mvpcom/carlaILTrainer#2]
Dear mvpcom,

I try to reimplement conditonal imitaition learning via keras. There is one problem which is that when testing the car most of time will make turning earlier. Do you have some suggestions for solving this problem? 

In validation data final average mean squared error for steer and brake is 0.077, that's best validation loss I can get ...

THanks a lot!

**Comment by mvpcom** [comment]
Hey @jackspp 

Is it possible to share your Keras implementation? I can't help you if I can't see the code. You can even pull a request to add your Keras branch to this repository.  There are many reasons for such a problem. It is not possible to tell about the exact problem right based on 0.077 and the current information. In my experiences, in some case it is bad, and in some case, it can be fair.
